apps/backend/src/users/users.service.ts

```diff
@@ -41,4 +35,37 @@ export class UsersService {
 
     return users;
   }
+
+  async findOne(userId: number) {
+    const user = await this.usersRepository.findOneBy({ id: userId });
+
+    if (!user) {
+      throw new BadRequestException('User not found');
+    }
+
+    const currentUser = getCurrentUser();
+
+    const currentStatus = currentUser.status;
+    const targetStatus = user.status;
+    switch (currentStatus) {
+      case Status.RECRUITER:
+        if (targetStatus === Status.ADMIN) {
```

Comment:
❓Should recruiters not be able to see admin information? Was this a design decision that was made? It seems like this switch statement would allow members and alumni to see admin information but not recruiters

Reply:
Thanks for pointing this out! I actually made a connection table for each user type and will attach it to the ticket and fix the switch statement.